.github/workflows/codacy-coverage-reporter.yaml

```
@@ -0,0 +1,16 @@
+name: Codacy Coverage Reporter
+
+on: ["push"]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)